README.md

```diff
@@ -22,15 +22,17 @@ Refer to the [`podman push`](http://docs.podman.io/en/latest/markdown/podman-man
 
 | Input Name | Description | Default |
 | ---------- | ----------- | ------- |
-| image	| Name of the image you want to push. | **Required**
+| image	| Name of the image you want to push. Eg. `username/imagename` or `imagename` | **Required**
 | tags | The tag or tags of the image to push. For multiple tags, separate by a space. For example, `latest ${{ github.sha }}`. | `latest`
-| registry | URL of the registry to push the image to. Eg. `quay.io/<username>` | **Required**
+| registry | URL of the registry to push the image to. Eg. `quay.io` or `quay.io/username` | **Required**
```

Comment:
this isn't a URL so let's say "hostname and optional namespace to push the image to"

Reply:
See the note below about naming image and registry.